host.go

```diff
@@ -101,6 +101,15 @@ func (h *Host) isOp(conn sshd.Connection) bool {
 // Connect a specific Terminal to this host and its room.
 func (h *Host) Connect(term *sshd.Terminal) {
 	id := NewIdentity(term.Conn)
+
+	if h.auth.keynamesMode {
+		name := h.auth.Keyname(id.PublicKey())
+		if len(name) >0 {
+			id.SetName(name)
+			id.SetSymbol("✓")
```

Comment:
Same as https://github.com/shazow/ssh-chat/pull/423/files#diff-7717c52956f97079739dd32cd1d2c7a9ff80173a3895ac11d9e845d1b6c5ca41R417

We shouldn't need to hard-code symbols, but rather load them from the comment.